util/readPokemonsCSV.go

```diff
@@ -0,0 +1,40 @@
+package util
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/tamnguyenwizeline/ondemand-go-bootcamp/model"
+)
+
+func ReadCSVFile(filePath string) ([]model.Pokemon, error) {
```

Comment:
As a good practice whenever we export something it is suggested to add a comment to give information about what they do

example:
// ReadCSVFile function based on a CSV file path would map the data into Pokemon structures.
func ReadCSVFile(filePath string) ([]model.Pokemon, error) {

Reply:
Thanks @MigGrover for suggesting this, but for me I personally think that the name of the exported type/ function already explains it unless it is too complicated. So please let me keep it as it is

Reply:
If you download the Linter it will warn you about it, also making those comments contribute to the documentation of your code :)